### PR TITLE
Feat add addrbook

### PR DESCRIPTION
**Pull request by AntiTyping**

### CODE REVIEW

**Comment by agouin** [comment]
Please re-run `make generate manifests` to get them in sync

File: internal/fullnode/pvc_builder.go [hunk 1]
```diff
@@ -79,9 +85,23 @@ func pvcResources(crd *cosmosv1.CosmosFullNode) corev1.ResourceRequirements {
...
+		//requestSize, unableToConvert := autoScale.RequestedSize.AsInt64()
+		//if unableToConvert == true {
+		//	fmt.Errorf("Unable to convert auto scale request")
+		//} else {
+		//	sizeWithPadding := resource.NewQuantity(int64(float64(requestSize)*SnapshotScalingFactor), resource.DecimalSI)
+		//	if sizeWithPadding.Cmp(size) > 0 {
+		//		reqs.Requests[corev1.ResourceStorage] = *sizeWithPadding
+		//	}
+		//}
+		requestedSize := autoScale.RequestedSize.DeepCopy()
```
Comment (agouin) [inline internal/fullnode/pvc_builder.go:100]: was this intentionally added to this branch? It appears unrelated.

Comment (AntiTyping) [reply internal/fullnode/pvc_builder.go:100]: probably rebase on the other branch I was working on. I will rebase.

**Review by agouin — APPROVED** [review]
